LiteCore/Storage/UnicodeCollator.cc

```diff
@@ -22,12 +22,20 @@
 #include "StringUtil.hh"
 #include <sqlite3.h>
 #include <algorithm>
+#define CBL_MATCH             0
+#define CBL_NOMATCH           1
+#define CBL_NOWILDCARDMATCH   2
```

Comment:
These should be declared in the header, since they're used as the return value.
Also, it'd be better to give them CamelCase names and put them in an enum.